utils/prefect.py

```diff
@@ -58,26 +59,45 @@ def get_flow_run_name(flow_run_id):
     return asyncio.run(_get_name(flow_run_id))
 
 
-async def _flow_run_query(tags, flow_run_name=None):
-    flow_runs_by_name = []
+async def _flow_run_query(
+    tags=None, flow_run_name=None, parent_flow_run_id=None, sort="START_TIME_DESC"
```

Comment:
I do not think we will ever use the function in this way, but what happens if all parameters are `None`, do we filter nothing and retrieve all flows?

Reply:
It retrieves all flows, I just did a quick test at my end.

Reply:
Ok, good to know!